suricata.yaml.in

```diff
@@ -556,6 +560,17 @@ outputs:
 # Logging configuration.  This is not about logging IDS alerts/events, but
 # output about what Suricata is doing, like startup messages, errors, etc.
 logging:
+
+  # The flush-interval governs how often Suricata will instruct the detection
+  # threads to flush their EVE output. Specify the value in seconds [1-60]
+  # and Suricata will initiate EVE log output flushes at that interval. A value
+  # of 0 means no EVE log output flushes are initiated. When the EVE output
+  # buffer-size value is non-zero, some EVE output that was written may remain
+  # buffered. The flush-interval governs how much buffered data exists.
+  #
+  # The default value is: 0 (never instruct detection threads to flush output)
+  #flush-interval: 0
```

Comment:
I don't think this belongs here since this section is specifically not about EVE logging.

Reply:
I struggled with placement for this as there is only one log flush thread and the interval is applied to it.

Can you suggest a more appropriate placement? Since it's global, including it in the `outputs` section for each doesn't seem to capture the global nature.

Reply:
> Since it's global, including it in the `outputs` section for each doesn't seem to capture the global nature.

Is it global to eve, or to stuff like fast-log, tls-log and http-log as well?

Reply:
Ideally `outputs` would not be an array so it could be a key there.  Another option is to put it under `eve-log` with the first occurrence setting the value, subsequent entries of `eve-log` would not reset that value.

Reply:
Or perhaps a top level `outputs-flush-interval`?

Reply:
I think we've had other possible use cases for a "heartbeat" feature. Should this be something like that? Then we could perhaps group things under that.

Reply:
@jasonish 
> Is it global to eve, or to stuff like fast-log, tls-log and http-log as well?
Just eve logs

@victorjulien  This works for me:
```
heartbeat:
   output_flush_interval: 
```